scripts/update_stale_work_references.py

```diff
@@ -11,13 +11,11 @@
 import datetime
 
 
-def main(ol_config: str, start_offset=0, days=31):
+def main(ol_config: str, days=1, skip=7):
```

Comment:
Note: This cron runs at 23:00 every day, so it makes sense for days here to be `1`. So it'll do one day, a week ago.